dashboard/app/views/levels/edit.html.haml

```diff
@@ -1,133 +1,137 @@
-%h1= @page_title = @level.new_record? ? "New #{@level.class.to_s} level" : "Editing #{@level.name}"
+%div{style: "display: flex; gap: 20px;"}
+  %div{style: "flex: 0 auto; min-width: 18%;", id: "table-of-contents-mount-point"}
+  = render partial: 'levels/editors/navigation_bar'
+  %div{style: "flex: 1;"}
```

Comment:
Much easier to review with the `Hide whitespace` option checked (i.e. `&w=1` in the URL)! :)